x-pack/plugins/security_solution/public/common/components/exceptions/types.ts

```diff
@@ -104,3 +107,32 @@ export type CreateExceptionListItemBuilderSchema = Omit<
 export type ExceptionsBuilderExceptionItem =
   | ExceptionListItemBuilderSchema
   | CreateExceptionListItemBuilderSchema;
+
+export interface FlattenedCodeSignature {
+  subject_name: string;
+  trusted: string;
+}
+
+export type FlattenType<T> = {
```

Comment:
Since typescript types are declarative, and since `type` could be inferred when working with a generic type, perhaps `Flattened` would be a better name